.github/workflows/ci.yml

```diff
@@ -21,8 +21,8 @@ jobs:
         distribution: 'adopt'
         cache: maven
     - name: Build azure-spring-boot-samples-4.x with Maven - Java ${{ matrix.java-version }}
-      run: mvn -f azure-spring-boot-samples-4.x-pom.xml --batch-mode --update-snapshots verify
+      run: mvn -f pom.xml -P spring-cloud-azure-4.x --batch-mode --update-snapshots verify
```

Comment:
Is it possible to delete `-P spring-cloud-azure-4.x` by using `active by default`?
Refs: https://maven.apache.org/guides/introduction/introduction-to-profiles.html